test/tests/integration/adoption-token.test.ts

```diff
@@ -69,6 +70,7 @@ describe('Adoption token', () => {
 
 		await addFakeProbe({ 'api:connect:adoption': adoptionStatusStub }, { query: { adoptionToken: 'adoptionTokenValue' } });
 
+		await setTimeout(100);
```

Comment:
`api:connect:adoption` happens asynchronously without being awaited so these were not reliable.